Makefile

```diff
@@ -40,6 +44,14 @@ fmt:
 vet:
 	go vet ./pkg/... ./cmd/...
 
+# Run go lint against code
+lint: 
+	golint ./pkg/... ./cmd/...
+
+# Run go imports against code
+imports:
+	goimports -w ./pkg/ ./cmd/
```

Comment:
Should this be used for checking a diff instead of writing out if it's for testing?